Cargo.toml

```diff
@@ -55,6 +55,7 @@ toml = "0.4.2"
 url = "1.1"
 clap = "2.31.2"
 unicode-width = "0.1.5"
+openssl = { version = '0.10.11', optional = true }
```

Comment:
Does cargo currently pull in openssl itself, or just openssl-sys? You can depend on openssl-sys 0.9.35 and use its vendored feature alternatively.

Reply:
Looks like Cargo depends on `crypto-hash` which depends on `openssl`-the-crate, although `openssl-sys` is less likely to change over time so it may be best to depend on that anyway